include/sisl/fds/buffer.hpp

```diff
@@ -224,6 +237,9 @@ class aligned_shared_ptr : public std::shared_ptr< T > {
     aligned_shared_ptr(T* p) : std::shared_ptr< T >(p) {}
 };
 
+struct io_blob;
+using io_blob_list_t = folly::small_vector< sisl::io_blob, 4 >;
```

Comment:
Does `folly::small_vector<T, N>` only need a forward declaration of `io_blob`? I think it probably depends on the entire declaration so it can expand and create default constructed members right?

Reply:
Interesting, I am not sure but I included this version of sisl (local build) in HomeReplication and used it and it compiles fine.

Reply:
What I'm saying is I don't think the forward declaration is doing anything is it? If it's removed does it failed to build and how...